state/indexer/block/kv/util.go

```diff
@@ -135,7 +136,7 @@ func parseEventSeqFromEventKey(key []byte) (int64, error) {
 func lookForHeight(conditions []query.Condition) (int64, bool, int) {
 	for i, c := range conditions {
 		if c.CompositeKey == types.BlockHeightKey && c.Op == query.OpEqual {
-			return c.Operand.(int64), true, i
+			return c.Operand.(*big.Int).Int64(), true, i
```

Comment:
This change in not present in `v0.37.x`, do we know why?

Reply:
Because  the function `lookForHeight` is not used at all, we use only `dedupHeight`. In 0.34.x we still keep the old behaviour hence using this function.